howso/direct/client.py

```diff
@@ -1224,18 +1331,20 @@ def copy_trainee(
                 f'binaries or camls, or a Trainee "{new_trainee_name}" already exists.')
 
         # Create the copy trainee
-        new_trainee = deepcopy(original_trainee)
+        new_trainee = DirectTrainee.from_dict(original_trainee.to_dict())
         new_trainee.name = new_trainee_name
         new_trainee._id = new_trainee_id  # type: ignore
```

Comment:
The user specified persistence value is not captured in the new_trainee
```suggestion
        new_trainee = DirectTrainee.from_dict(dict(
            original_trainee.to_dict(),
            id=new_trainee_id,
            name=new_trainee_name,
            persistence=persistence
        ))
```